### PR TITLE
Update generated swagger.json for client API v6

### PR DESCRIPTION
**Pull request by akshaymankar**
https://wearezeta.atlassian.net/browse/WPB-10737

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: changelog.d/1-api-changes/finalise-v6 [hunk 1]
```diff
@@ -1 +1 @@
-Finalise version 6 and introduce new development version 7
+Finalise version 6 and introduce new development version 7 (#4179, ##)
```
Comment (supersven) [inline changelog.d/1-api-changes/finalise-v6:1]:
```suggestion
Finalise version 6 and introduce new development version 7 (#4179)
```

Or, does `##` have a special meaning? :thinking:

Comment (akshaymankar) [reply changelog.d/1-api-changes/finalise-v6:1]: ## will put this PR's number when I squash it.

Comment (akshaymankar) [reply changelog.d/1-api-changes/finalise-v6:1]: Documented here: https://docs.wire.com/developer/developer/pr-guidelines.html#changelog-entries

File: services/brig/docs/swagger-v6.json [hunk 2]
```diff
@@ -3321,16 +3329,16 @@
       "MLSKeys": {
         "properties": {
           "ecdsa_secp256r1_sha256": {
-            "$ref": "#/components/schemas/MLSPublicKey"
+            "$ref": "#/components/schemas/SomeKey"
```
Comment (stefanwire) [inline services/brig/docs/swagger-v6.json:3332]: This will be different for v6 on q1, I think. Can have a look later.

**Review by stefanwire — APPROVED** [review]

**Pull request merged**